config/drupal/phpcsfixer.rules.yml

```diff
@@ -68,7 +69,7 @@ parameters:
         indentation_type: false
         line_ending: true
         lowercase_keywords: true
-        class_attributes_separation: false
+        class_attributes_separation: true
```

Comment:
After digging a little bit I see we change this from `false` => `true`. 

do we really need to change it ? does it comply with phpcs (drupal/coder) ?

Reply:
I think I changed it to comply with Coder.

I had no problem with it enabled.

But if problematic let's turn it to false again.

Reply:
after some tests, having enabled `class_attributes_separation`does not make this config more compliant with phpcs (drupal/coder). I mean, phpcs (drupal/coder) will update the class indeed with whitespaces between attributes, and having `class_attributes_separation: false` does not revert the changes.

So having it unchanged (`class_attributes_separation: false`) will for sure not have side effects.
But having it enabled (`class_attributes_separation: true`) is ineed more logical but may have side effects in the future if php-cs-fixer change how `class_attributes_separation: true` react and make it un-compatible with phpcs (drupal/coder) rules.

I would say that we should enable it and monitor it - maybe one day we will need to disable it leaving this changes to phpcs (drupal/coder).